.github/workflows/ci-integration-tests-gcp.yml

```diff
@@ -101,20 +101,20 @@ jobs:
   # The default network is mainnet unless a manually triggered workflow or repository variable
   # is configured differently.
   #
-  # The outputs for this job have the same names as the workflow outputs in find-cached-disks.yml
+  # The outputs for this job have the same names as the workflow outputs in manual-find-cached-disks.yml
   get-available-disks:
     name: Check if cached state disks exist for ${{ inputs.network || vars.ZCASH_NETWORK }}
-    uses: ./.github/workflows/find-cached-disks.yml
+    uses: ./.github/workflows/manual-find-cached-disks.yml
```

Comment:
manual-find-cached-disks.yml is a sub-workflow, not a manual workflow.

Reply:
Fixed with [chore: use better name for find-disks](https://github.com/ZcashFoundation/zebra/pull/7616/commits/3db9e6c89733c5baf6c573a3d555a4f8c9178016)